### PR TITLE
Feature/home

### PR DESCRIPTION
**Pull request by iortega-dev**

### CODE REVIEW

File: src/screens/home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        </Link>
+      </Grid>
+      <Grid item xs={12} sm={12} md={6} lg={4}>
+        <Link href="/locations">
+          <PaperCard type={PaperCardType.Location}/>
+        </Link>
+      </Grid>
+    </Grid>
+  )
+}
```
Comment (JaimeJesusSerrano) [inline src/screens/home/index.tsx]: A newline is required here

File: src/screens/home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Grid, Link } from '@material-ui/core'
+import { useLocation } from 'react-router-dom'
+import { PaperCard, PaperCardType } from './components/PaperCard'
+
+export default () => {
```
Comment (JaimeJesusSerrano) [inline src/screens/home/index.tsx]: We use to add a component Name (Home in this case) and after export it.

**Review by JaimeJesusSerrano — CHANGES_REQUESTED** [review]
I had commented two small changes, please review them.

File: src/screens/home/index.tsx (hunk 1 above)
Comment (iortega-dev) [reply src/screens/home/index.tsx]: Done

File: src/screens/home/index.tsx (hunk 2 above)
Comment (iortega-dev) [reply src/screens/home/index.tsx]: Done

**Review by JaimeJesusSerrano — APPROVED** [review]

**Pull request merged**